rest/src/test/resources/db/migration/V202010042030__baseline.sql

```diff
@@ -0,0 +1,125 @@
+CREATE TABLE ATTRIBUTE
```

Comment:
Is this a new file structure? And if yes, is there a method to migrate the old data to the new structure?